### PR TITLE
Add variant Outlined to Button.ts

### PR DESCRIPTION
**Pull request by murilopereirame**

### CODE REVIEW

**Review by charlag — COMMENTED** [review]
After we refactored buttons we tried really hard to cut down on variants of normal `Button`. Are you sure it's best to extend it again instead of making another one?

File: src/common/gui/main-styles.ts [hunk 1]
```diff
@@ -2641,5 +2641,13 @@ styles.registerStyle("main", () => {
 		".single-line-text-field::placeholder": {
 			color: theme.content_message_bg,
 		},
+		".tutaui-button-outline": {
```
Comment (charlag) [inline src/common/gui/main-styles.ts:2644]: we also try to not add classes for components and use utility classes instead if we can help it

File: src/common/gui/base/buttons/OutlineButton.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		return m(BaseButton, {
+			label: lang.getMaybeLazy(attrs.label),
+			text: attrs.text ? lang.getMaybeLazy(attrs.text) : lang.getMaybeLazy(attrs.label),
+			onclick: attrs.click,
+			disabled: attrs.disabled,
+			style: {
+				borderColor: theme.content_message_bg,
+				color: theme.content_button,
+			},
+			class: this.resolveClasses(attrs.expanded, attrs.disabled).join(" "),
```
Comment (murilopereirame) [inline src/common/gui/base/buttons/OutlineButton.ts:36]: You can call the join inside the function and return just the string, I think it's a bit more clean

File: src/common/gui/base/Button.ts [hunk 3]
```diff
@@ -21,65 +21,42 @@ export const enum ButtonColor {
 	Elevated = "elevated",
 	DrawerNav = "drawernav",
 	Fab = "fab",
+	Neutral = "neutral",
```
Comment (murilopereirame) [inline src/common/gui/base/Button.ts:24]: This can be removed

**Pull request merged**